src/main/java/com/hrznstudio/titanium/Titanium.java

```diff
@@ -82,12 +94,28 @@ public Titanium() {
         EventManager.mod(FMLCommonSetupEvent.class).process(this::commonSetup).subscribe();
         EventManager.forge(PlayerEvent.PlayerLoggedInEvent.class).process(this::onPlayerLoggedIn).subscribe();
         EventManager.forge(FMLServerStartingEvent.class).process(this::onServerStart).subscribe();
+
     }
 
     public static void openGui(TileActive tile, ServerPlayerEntity player) {
         NetworkHooks.openGui(player, tile, tile.getPos());
     }
 
+    @Override
+    public void onPreInit() {
+        super.onPreInit();
+        ResourceRegistry.scanForReferences();
+        ResourceRegistry.getOrCreate("iron").withOverride(ResourceType.ORE, Blocks.IRON_ORE).withOverride(ResourceType.METAL_BLOCK, Blocks.IRON_BLOCK).withOverride(ResourceType.INGOT, Items.IRON_INGOT).withOverride(ResourceType.NUGGET, Items.IRON_NUGGET);
+        ResourceRegistry.getOrCreate("gold").withOverride(ResourceType.ORE, Blocks.GOLD_ORE).withOverride(ResourceType.METAL_BLOCK, Blocks.GOLD_BLOCK).withOverride(ResourceType.INGOT, Items.GOLD_INGOT).withOverride(ResourceType.NUGGET, Items.GOLD_NUGGET);
+        ResourceRegistry.getOrCreate("coal").withOverride(ResourceType.ORE, Blocks.COAL_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.COAL_BLOCK).withOverride(ResourceType.GEM, Items.COAL);
+        ResourceRegistry.getOrCreate("lapis_lazuli").withOverride(ResourceType.ORE, Blocks.LAPIS_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.LAPIS_BLOCK).withOverride(ResourceType.GEM, Items.LAPIS_LAZULI);
+        ResourceRegistry.getOrCreate("diamond").withOverride(ResourceType.ORE, Blocks.DIAMOND_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.DIAMOND_BLOCK).withOverride(ResourceType.GEM, Items.DIAMOND);
+        ResourceRegistry.getOrCreate("redstone").withOverride(ResourceType.ORE, Blocks.REDSTONE_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.REDSTONE_BLOCK).withOverride(ResourceType.DUST, Items.REDSTONE);
+        ResourceRegistry.getOrCreate("emerald").withOverride(ResourceType.ORE, Blocks.EMERALD_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.EMERALD_BLOCK).withOverride(ResourceType.GEM, Items.EMERALD);
+        ResourceRegistry.getOrCreate("nether_quartz").withOverride(ResourceType.NETHER_ORE, Blocks.NETHER_QUARTZ_ORE).withOverride(ResourceType.GEM_BLOCK, Blocks.QUARTZ_BLOCK).withOverride(ResourceType.GEM, Items.QUARTZ);
+        ResourceRegistry.getOrCreate("glowstone").withOverride(ResourceType.GEM_BLOCK, Blocks.GLOWSTONE).withOverride(ResourceType.DUST, Items.GLOWSTONE_DUST);
```

Comment:
Should titanium really do this? what if a mod isnt going to use it, isnt it just wasteful?

Reply:
Yes. What is the point of having the system if you are going to have duplicates with minecraft stuff.

Reply:
isnt that the entire point of the resource registry tho? so it doesnt duplicate?

Reply:
That's what are the overrides are for, so you can add mc things by default